src/components/Table/DelegatesTable.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Table } from 'decentraland-ui/dist/components/Table/Table'
+
+import { Delegate } from '../../hooks/useDelegatesInfo'
+import { useSortingByKey } from '../../hooks/useSortingByKey'
+import Sort from '../Icon/Sort'
+import DelegateRow from '../Table/DelegatesTableRow'
+
+import './DelegatesTable.css'
+
+const DISPLAYED_CANDIDATES = 9
+const DEFAULT_SORTING_KEY: DelegateKey = 'lastVoted'
+type DelegateKey = keyof Delegate
+
+interface Props {
+  delegates: Delegate[]
+}
+
+const DelegatesTable = ({ delegates }: Props) => {
+  const t = useFormatMessage()
+  const [sortingKey, setSortingKey] = useState(DEFAULT_SORTING_KEY as DelegateKey)
+  const { sorted: delegatesSorted, changeSort, isDescendingSort } = useSortingByKey(delegates, sortingKey)
+
+  const updateSort = (newSortingKey: DelegateKey) => {
+    if (newSortingKey !== sortingKey) {
+      setSortingKey(newSortingKey)
+    } else {
+      changeSort()
+    }
+  }
+
+  return (
+    <>
+      <Table basic="very" className="DelegatesTable">
+        <Table.Header className="DelegatesTable__Header">
+          <Table.Row>
+            <Table.HeaderCell>{t('page.home.dao_delegates.candidate_name')}</Table.HeaderCell>
+            <Table.HeaderCell onClick={() => updateSort('lastVoted')}>
+              <span>
+                {t('page.home.dao_delegates.last_voted')}
+                <Sort descending={isDescendingSort} selected={sortingKey === 'lastVoted'} />
+              </span>
+            </Table.HeaderCell>
+            <Table.HeaderCell onClick={() => updateSort('timesVoted')}>
+              <span>
+                {t('page.home.dao_delegates.times_voted')}
+                <Sort descending={isDescendingSort} selected={sortingKey === 'timesVoted'} />
+              </span>
+            </Table.HeaderCell>
+            <Table.HeaderCell onClick={() => updateSort('pickedBy')}>
+              <span>
+                {t('page.home.dao_delegates.picked_by')}
+                <Sort descending={isDescendingSort} selected={sortingKey === 'pickedBy'} />
+              </span>
+            </Table.HeaderCell>
+            <Table.HeaderCell onClick={() => updateSort('totalVP')}>
+              <span>
+                {t('page.home.dao_delegates.total_vp')}
+                <Sort descending={isDescendingSort} selected={sortingKey === 'totalVP'} />
```

Comment:
I don't know why I could not pass `isDescendingSort` renamed as `descending` directly to the Sort component, and had to do it explicitly.